### ISSUE TITLE
[Feature suggestion]: auot-switch modi [dia-show style]

### ISSUE THREAD

**Issue by sebo505** [NickvisionApps/Cavalier#156]
You can change modi by pressing d on keyboard.  I think it would be a handy feature to add a dia-show modus where you can set a time in seconds after which it will forward to the next modus automatically, maybe even shuffle modi.

I have achieved this for me via a bash script wich is kind of hacky and prone to break.
Maybe you are interested in adding such a feature natively.

**Comment by RageGamerBoi** [comment]
Do you mean "modes"?

**Comment by sebo505** [comment]
Yes, modes. I don't know if this is a feature that makes sense to anyone but me. I personally like having cavalier maximized and than have it changed modes automatically while listen to music. 
In X, I do this via a bash script that checks whether cavalier window has focus and the use xdotool to press d every x seconds if window has focus.

https://github.com/NickvisionApps/Cavalier/assets/65250211/41a5ff5a-a3e2-40c8-816a-fb5242e56e35

**Comment by buchpat** [comment]
that would be nice, also like a visual playlist.

i'm running cavalier on my tiny side monitor if music is running, cavalier is really cool :heart:
I also found it useful since it helps prevent screen burn-in. :)

![Image](https://github.com/user-attachments/assets/06abd42e-e33e-4440-80a1-d403357546bd)
